src/style.css

```diff
@@ -0,0 +1,28 @@
+.starts {
+  display: flex;
+}
+
+.stars__star {
+  width: 50px;
+  height: 50px;
+  background-image: url(images/star.svg);
+  margin: 0;
+  flex: 1;
+}
+
+.body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+}
+
+.stars--0 .stars__star:nth-child(1),
+.stars--1 .stars__star:nth-child(2),
+.stars--2 .stars__star:nth-child(3),
+.stars--3 .stars__star:nth-child(4),
+.stars--4 .stars__star:nth-child(5),
+.stars--5 .stars__star:nth-child(5) {
+  background-image: url("images/star-active.svg");
+}
```

Comment:
You shouldn't target explicitly i.e. `:nth-child(4)`.  Instead use functional notation (like `-n + 4`). Please Look for it [in the docs](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child).